lib/twitter_cldr/js/mustache/data_bundle.coffee

```diff
@@ -0,0 +1,38 @@
+###
+// Copyright 2012 Twitter, Inc
+// http://www.apache.org/licenses/LICENSE-2.0
+// TwitterCLDR (JavaScript) v{{version}}
+// Authors:     Cameron Dutro [@camertron]
+                Kirill Lashuk [@KL_7]
+                portions by Sven Fuchs [@svenfuchs]
+// Homepage:    https://twitter.com
+// Description: Provides date, time, number, and list formatting functionality for various Twitter-supported locales in Javascript.
+###
+
+{{#source_map}}
+###
+//@ sourceMappingURL={{locale}}.map
+###
+{{/source_map}}
+
+# required alias definition that adds twitter_cldr to Twitter's static build process
+###<<module_def>>###
+
+
+TwitterCldrDataBundle =
+{{{contents}}}
+
+root = if exports?
+  exports
+else
+  this.TwitterCldrDataBundle = {}
```

Comment:
Hm, after thinking a bit more about it, with what I'm suggesting we'd either have to silently override `TwitterCldr.data` in the data bundle or it'd be impossible to reload the data bundle, because it won't override what's already loaded and we won't have a separate global reference for the newly loaded data. 

I don't like either of the alternatives, so I guess we should keep `TwitterCldrDataBundle`, at least for now.

On the other hand (correct me if I'm wrong), even with `TwitterCldrDataBundle` it's impossible to simultaneously load more than one locale bundle and dynamically switch between them. I'm not sure there's a use case for that, but it'd be an interesting problem to tackle. I'll probably work on it a bit later, but for this PR let's just go with two separate global objects.